tests/components/Header.test.tsx

```diff
@@ -1,9 +1,9 @@
-import Hello from '../../src/components/Hello';
+import Header from '../../src/components/Header';
 import { render } from '@testing-library/react';
 
-describe('Hello', () => {
```

Comment:
please, add some more testing cases. Validate that the name renders as expected, for instance

Reply:
Done!